cmd/crio/config.go

```diff
@@ -188,6 +188,9 @@ read_only = {{ .ReadOnly }}
 # are fatal, panic, error, warn, info, and debug.
 log_level = "{{ .LogLevel }}"
 
+# The default log directory were all logs will go unless directly specified by the kubelet
```

Comment:
nit of a nit s/were/where/  Don't worry about it if you don't have other changes.

Reply:
Ah this copy/paste error, fixed it.